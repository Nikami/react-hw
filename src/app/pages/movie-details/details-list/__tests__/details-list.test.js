import React from 'react';
import { shallow } from 'enzyme';

import DetailsList from '../index';
import DetailsFilter from '../../details-filter';
import MovieList from '../../../shared/components/movie-list';

describe('DetailsList', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<DetailsList />);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('has DetailsFilter', () => {
    expect(wrapper.find(DetailsFilter).length).toBe(1);
  });

  it('has MovieList', () => {
    expect(wrapper.find(MovieList).length).toBe(1);
  });
});
