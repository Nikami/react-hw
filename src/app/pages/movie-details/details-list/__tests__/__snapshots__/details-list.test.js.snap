// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsList matches snapshot 1`] = `
<Fragment>
  <_default />
  <WithStyles(MovieList)
    movies={Array []}
  />
</Fragment>
`;
